src/App.js

```diff
@@ -1,10 +1,40 @@
+import { useState } from 'react';
+import Container from './components/Container';
+
 function App() {
 
+  const [numbers, setNumbers] = useState([]);
+  const [numbersList, setNumbersList] = useState([]);
+
+  function getUniqueRandomNumber(array, limitNumber) {
+      let uniqueRandomNumber;
+
+      function getNumber() {
+          const randomNumber = Math.floor((Math.random() * limitNumber) + 1);
+          getUniqueNumber(randomNumber);
```

Comment:
skoro masz funkcje getNumber, to po co jeszcze getUniqueNumber? Zaimplementuj instrukcje warunkową z wykorzystaniem tylko getNumber i przekazanej tablicy by sprawdzić czy liczba jest unikalna

Reply:
Przeniosłem logikę do pętli while:
`  function getUniqueRandomNumber(array, limitNumber) {
    let uniqueRandomNumber;

    while (uniqueRandomNumber === undefined) {
      const randomNumber = Math.floor((Math.random() * limitNumber) + 1);
      const uniqueNumber = !(array.includes(randomNumber));
      if (uniqueNumber) {
        return uniqueRandomNumber = randomNumber;
      };
    };
    return uniqueRandomNumber;
  }`

Reply:
Przerobiłem logikę aby zamiast funkcji wykorzystywana była pętla while:

`
  function getUniqueRandomNumber(array, limitNumber) {
    let uniqueRandomNumber;

    while (uniqueRandomNumber === undefined) {
      const randomNumber = Math.floor((Math.random() * limitNumber) + 1);
      const uniqueNumber = !(array.includes(randomNumber));
      if (uniqueNumber) {
        return uniqueRandomNumber = randomNumber;
      };
    };
    return uniqueRandomNumber;
  }`